cmd/cgr-tester/cgr-tester.go

```diff
@@ -270,49 +274,68 @@ func main() {
 		return
 	}
 
-	var timeparsed time.Duration
-	var err error
-	tstart := time.Now()
-	timeparsed, err = utils.ParseDurationWithNanosecs(*usage)
-	if err != nil {
-		return
-	}
-	tend := tstart.Add(timeparsed)
-	cd := &engine.CallDescriptorWithAPIOpts{
-		CallDescriptor: &engine.CallDescriptor{
-			TimeStart:     tstart,
-			TimeEnd:       tend,
-			DurationIndex: 60 * time.Second,
-			ToR:           *tor,
-			Category:      *category,
-			Tenant:        *tenant,
-			Subject:       *subject,
-			Destination:   *destination,
-		},
-	}
-	var duration time.Duration
-	if len(*raterAddress) == 0 {
-		duration, err = durInternalRater(cd)
-	} else {
-		duration, err = durRemoteRater(cd)
-	}
-	if err != nil {
-		log.Fatal(err.Error())
-	} else {
-		log.Printf("Elapsed: %s resulted: %f req/s.", duration, float64(*runs)/duration.Seconds())
-	}
+	if exec != nil && *exec != utils.EmptyString {
+		tasks := strings.Split(*exec, utils.FieldsSep)
+		for _, taskID := range tasks {
```

Comment:
here we do not need multiple tasks since we would only use cgr-tester to profile one command (exec)